### ISSUE TITLE
FOLLOW YOUR WORK

### ISSUE THREAD

**Issue by shz1314** [KupynOrest/DeblurGAN#57]
hi,
   I have done a little work follow your Deblur GAN work, but when I train your code which i get from github the result on the GoPRO dataset is not good. Just PSNR=26.02 and SSIM=0.88222. So can you give me your train peremeter on the GoPRO dataset, such as the learning rate. In your paper. you say 150 epoch pretrain, but in your code the learingrate decay is null. So i just train 300 epoch. Does it make the bad results? And can you give me your final weights on G. I will do the contrast test. My email is 
haoze_song_jnu@163.com . So thanks.
                                                                                                                                                                                                                    Yours,
                                                                                                                                                                                                                     Haoze

**Comment by KupynOrest** [comment]
Many thanks for your interest, I Sent a message to your e-mail.
